.github/pull_request_template.md

```diff
@@ -12,11 +12,17 @@
 
 
 ## Checklist:
-<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+
+<!--- Our Continuous Integration Workflow will automatically check if all unit tests run without failure -->
+<!--- Our Continuous Integration Workflow will automatically check if the code coverage decreases -->
+<!--- The following tasks must be performed manually by the contributor and checked by the reviewer -->
+
+<!--- Go over all the following points, and put an `x` in all the boxes in the "Finished" column that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] A test for the new functionality was added.
-- [ ] All tests run without failure.
-- [ ] The new code complies with the TiGL style guide.
-- [ ] New classes have been added to the Python interface.
-- [ ] API changes were documented properly in tigl.h.
-- [ ] Changes were documented in tigl/ChangeLog.md
+
+| Task                                                    | Finished                                                  | Reviewer Approved          |
+|---------------------------------------------------------|-----------------------------------------------------------|----------------------------|
+| At least test for the new functionality was added.      | <ul><li>- [ ] yes </li><li>- [ ] does not apply</li></ul> | <ul><li>- [ ] OK</li></ul> |
```

Comment:
Is there an 'one' missing? Or what do you mean by 'at least'?

Reply:
Yes, I fixed it. Thanks.